compiler/testData/codegen/boxInline/smap/inlineOnly/stdlibInlineOnlyOneLine.kt

```diff
@@ -1,11 +1,30 @@
+// FILE: 1.kt
+package test
+
 @Suppress("INVISIBLE_MEMBER", "INVISIBLE_REFERENCE")
 @kotlin.internal.InlineOnly
 inline fun <T, R> T.myLet(block: (T) -> R) = block(this)
 
+// FILE: 2.kt
+import test.*
+
 fun box(): String {
     val k = "".myLet { it + "K" }
     return "O".myLet(fun (it: String): String { return it + k })
 }
 
+// FILE: 2.smap
 // See KT-23064 for the problem and InlineOnlySmapSkipper for an explanation.
-// 2 LINENUMBER 65100
+SMAP
+2.kt
+Kotlin
+*S Kotlin
+*F
++ 1 2.kt
+_2Kt
++ 2 fake.kt
+kotlin/jvm/internal/FakeKt
+*L
+1#1,9:1
+65100#2:10
```

Comment:
cc @yanex Could this approach be used for coroutines?

Do we still need here to use 65100? "1" looks more intuitive and require less space

Reply:
I've implemented it in https://github.com/JetBrains/kotlin/commit/f5affff6a4add981c1ba87857c9780c1ee7a1efd

Reply:
I did it a bit differently to avoid a magic constant 1 and to change existing instances of 65100 to 1 until the standard library is rebootstrapped: https://github.com/JetBrains/kotlin/pull/3210/commits/cf2ca80c4ba1c28c89374d5b7ed4354c3dd11c03

Reply:
@pyos Thank you, actually it's a bug in my implementation with missed remapping